test/ini_files/cgap_alpha.ini

```diff
@@ -20,7 +20,10 @@ s3_encrypt_key_id = ${S3_ENCRYPT_KEY_ID}
 accession_factory = encoded.server_defaults.enc_accession
 elasticsearch.server = search-fourfront-cgap-ewf7r7u2nq3xkgyozdhns4bkni.us-east-1.es.amazonaws.com:80
 snovault.app_version = ask-pip
+env.bucket = foursight-envs
```

Comment:
Not a sensible fallback for CGAP I'd say... Not that a good one exists. If this is blank will it throw an error? This bucket should always be available when the ECS comes up so I think it's okay to require. Same comment in other cgap ini files.

Reply:
Well, this is just a sample file, so this isn't the place where the fallback is computed. I'll make an adjustment to the defaulting, though.